velox/functions/sparksql/tests/ArithmeticTest.cpp

```diff
@@ -146,6 +146,91 @@ class ArithmeticTest : public SparkFunctionBaseTest {
     return evaluateOnce<double>("divide(c0, c1)", numerator, denominator);
   }
 
+  template <typename T>
+  std::optional<T> checkedArithmetic(
```

Comment:
nit: it seems this method could be removed because it is simple and does not simply the call. We could follow the impl. of divide.
> return evaluateOnce<double>("divide(c0, c1)", numerator, denominator);